gnd/google-services.json

```diff
@@ -1,51 +0,0 @@
-{
```

Comment:
Please revert this one and remove the unused file [gnd_geojson.json](https://github.com/google/ground-android/blob/master/gnd/src/main/res/raw/gnd_geojson.json)

Reply:
80f43c4 sorry about that!